docs/source/guides/build.rst

```diff
@@ -250,25 +232,19 @@ directory of your choice.
 Building Iroha
 ^^^^^^^^^^^^^^
 
-Building on Windows differs from the main flow and the guide is `here <#building-iroha-on-windows>`_.
-
 To build Iroha, use those commands
 
 .. code-block:: shell
 
-  mkdir build; cd build; cmake ..; make -j$(nproc)
+  cmake -H. -Bbuild -DCMAKE_TOOLCHAIN_FILE=/path/to/vcpkg/scripts/buildsystems/vcpkg.cmake -G "Ninja"
+  cmake --build build
 
-Alternatively, you can use these shorthand parameters (they are not documented
-though)
+.. note:: When building on Windows do not execute this from the Power Shell. Better use x64 Native tools command prompt.
 
-.. code-block:: shell
-
-  cmake -H. -Bbuild;
-  cmake --build build -- -j$(nproc)
-
-.. note::  On macOS ``$(nproc)`` variable does not work. Check the number of
-  logical cores with ``sysctl -n hw.ncpu`` and put it explicitly in the command
-  above, e.g. ``cmake --build build -- -j4``
+.. note:: You can build in multiple threads to speed the process up by a command ``cmake --build build -- -j$(nproc)``
```

Comment:
To avoid multiple exceptions to this rule, I'd make it more general, like
```suggestion
.. note:: You can build in multiple threads to speed the process up by a command ``cmake --build build -- -j <number of threads>``
```
and then giving instructions on how to choose the number of threads depending on the number of processors and how to find the later on different platforms.